StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkPaymentController.swift

```diff
@@ -225,7 +225,7 @@ import UIKit
         }
 
         guard let linkAccountSession = linkAccountSession else {
-            continuation.resume(throwing: PaymentSheetError.unknown(debugDescription: "Failed to create link account session"))
+            continuation.resume(throwing: PaymentSheetError.failedToCreateLinkSession)
```

Comment:
nit (feel free to ignore): no need for "to" --  failedCreateLinkSession

Reply:
Personally like how it reads with the "to"